Convert md files to rst, split CONTRIBUTING.md (#5)
ansible/community-docs: contributing.rst

************
Contributing
************

> We follow `Ansible Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_ in all our contributions and interactions within this repository.

Reviewer: ```suggestion
We follow the `Ansible Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_ in all our contributions and interactions within this repository.
```

If you find any inconsistencies or places in this document which can be improved, please raise an issue or pull request to fix it.

If you are a committer, also refer to the `Ansible committer guidelines <https://docs.ansible.com/ansible/devel/community/committer_guidelines.html>`_.

Issue tracker
=============

Whether you are looking for an opportunity to contribute or you found a bug and already know how to solve it, please go to the collection's issue tracker first.
There you can find feature ideas to implement, reports about bugs to solve, or submit an issue to discuss your idea before implementing it which can help choose a right direction at the beginning of your work and potentially save a lot of time and effort.
Also somebody may already have started discussing or working on implementing the same or a similar idea,
so you can cooperate to create a better solution together.

> Many collections use ``easy_fix`` or a similarly named label to indicate issues that are a good starting point for beginners (you might also look at the `newbie pinboard <https://github.com/ansible/community/issues/437>`_ if you do not know which collection to start with). Also ``help_wanted`` or similarly named labels often indicate that this is a feature that is deemed (potentially) useful, but none of the maintainers want to invest time to actually implement it.

Reviewer: ```suggestion
Many collections use ``easy_fix`` or a similarly named label to indicate issues that are a good starting point for beginners (you might also look at the `newbie pinboard <https://github.com/ansible/community/issues/437>`_ if you do not know which collection to start with). Also ``help_wanted`` or similarly named labels often indicate that this is a feature that is deemed (potentially) useful, but none of the maintainers can invest time to actually implement it.
```

Author: hehe:) good spot, thanks. committed

Open pull requests
==================

Look through the currently open pull requests in the collection repository.
You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_.
And it is always worth saying that good reviews are often more valuable than pull requests themselves.

Also, consider taking up a valuable, reviewed, but abandoned pull request which you could politely ask the original authors to complete yourself.

Looking for an idea to implement
================================

First, see the paragraphs above.

If you came up with a new feature, it is always worth creating an issue

> before starting to write code to discuss the idea with the community first.

Reviewer: ```suggestion
to discuss the idea with the community first, before writing code.
```

Author: much better, thanks! committed

If you are going to implement the feature yourself, say it in the issue explicitly to avoid working in parallel with somebody else.

Create your first pull request
==============================

> How to quickly create your first patch and submit a pull request, refer to the `Quick-start guide <create_pr_quick_start_guide.rst>`_

Reviewer: ```suggestion
To learn how to quickly create your first patch and submit a pull request, refer to the `Quick-start guide <create_pr_quick_start_guide.rst>`_
```
